dask/array/creation.py

```diff
@@ -411,7 +411,7 @@ def indices(dimensions, dtype=int, chunks='auto'):
     return grid
 
 
-def eye(N, chunks, M=None, k=0, dtype=float):
+def eye(N, chunks='auto', M=None, k=0, dtype=float):
```

Comment:
Could you update the docstring for the `chunks` parameter to allow for values of type `str`

Reply:
Not sure the docstring update I added is clear enough :). Let me know if there's a better alternative.